### PR TITLE
feat: add the flag config Clear method

### PR DESCRIPTION
**Pull request by yuhao900914**
### Checklist

* [ ] Does your PR title have the correct [title format](https://github.com/amplitude/experiment-ios-client/blob/main/CONTRIBUTING.md#pr-commit-title-conventions)?
* Does your PR have a breaking change?:

### CODE REVIEW

File: Experiment.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -311,6 +312,27 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		19F2A56C28FDE17C002DF58F /* ShellScript */ = {
```
Comment (yuhao900914) [inline Experiment.xcodeproj/project.pbxproj:316]: This is to fix the build failed issue of `xframework that cannot be found`  after running `carthage update --use-xcframeworks`.
The changes are coming from adding the script `/usr/local/bin/carthage copy-frameworks
` under the Build Phases tab.
For reference. https://kevinle.medium.com/step-by-step-guide-on-using-carthage-dependency-manager-a29c15f9a1ac.

Comment (qingzhuozhen) [reply Experiment.xcodeproj/project.pbxproj:316]: @bgiori Can you confirm if we need this?

**Review by qingzhuozhen — APPROVED** [review]

File: Experiment.xcodeproj/project.pbxproj (hunk 1 above)
Comment (bgiori) [reply Experiment.xcodeproj/project.pbxproj:316]: I did a fresh clone of the repository, ran `carthage update --use-xcframeworks` opened the project in xcode and built it successfully without the build phase in question. 

What carthage version are you using? `carthage version`

Comment (yuhao900914) [reply Experiment.xcodeproj/project.pbxproj:316]: @bgiori  I'm using `0.38.0` how about yours?

Comment (bgiori) [reply Experiment.xcodeproj/project.pbxproj:316]: Yeah, same. Do you have the exact output for the error which this change fixes?

**Review by bgiori — APPROVED** [review]

**Pull request merged**